_lifeofinternatpuppet/post_41.md

```diff
@@ -0,0 +1,48 @@
+layout: page
+title: Week 41
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 41
+---
+
+Hi everyone!
+
+I had a lovely PTO last week! This week was busy as always with IAC Confluence Wiki Project.
+
+I closed the following tickets and pages were made live on Confluence
+- [IAC-1640](https://tickets.puppetlabs.com/browse/IAC-1640)
+The layout was changed for the 'Features/Tooling and Tasks' page and the colour coded column was removed before the ticket was closed.
+
+- [IAC-1641](https://tickets.puppetlabs.com/browse/IAC-1641)
+The action points for the page 'Modules Team Triage Process' were updated in the Google Doc and within the above ticket.
+
+- [IAC-1642](https://tickets.puppetlabs.com/browse/IAC-1642)
+- [IAC-1634](https://tickets.puppetlabs.com/browse/IAC-1634)
+
+- [IAC-1639](https://tickets.puppetlabs.com/browse/IAC-1639)
+When the above ticket IAC-1639 was under review, I got feedback from my mentor Paula and the new changes were made before closing the ticket.
+
+I also added another page link called 'Show and Tell Schedule' within the New Start Guide under the section called 'Useful Reading Materials'
+Check the new comments in this [ticket](https://tickets.puppetlabs.com/browse/IAC-1626) This ticket will be reopened again in the future to add some more links in the 'Useful Reading Materials' section.
+
+I have also raised a ticket with IT support to unarchive a page called 'un-approved process page' as my team didn't have sufficient permissions to do this action. The Google doc was also updated with the latest comments. The content of the [existing ticket](https://tickets.puppetlabs.com/browse/IAC-1644) was updated to reflect the new changes for example the title, acceptance criteria and description were updated. I will work on this ticket once the page is unarchived. The ticket raised to IT support was also shared with my manager Norman for tracking purposes.
```

Comment:
`The ticket raised to IT support was also shared with my manager Norman for tracking purposes.`
You can remove this comment I dont feel like its relevant to the blog post.

Reply:
The comment is removed @pmcmaw. Check the new commit message ⬇️